packages/ai/package.json

```diff
@@ -54,14 +57,19 @@
     }
   },
   "dependencies": {
+    "@ai-sdk/anthropic": "workspace:^",
     "@ai-sdk/gateway": "workspace:*",
+    "@ai-sdk/google": "workspace:^",
+    "@ai-sdk/groq": "workspace:^",
+    "@ai-sdk/openai": "workspace:^",
     "@ai-sdk/provider": "workspace:*",
     "@ai-sdk/provider-utils": "workspace:*",
-    "@opentelemetry/api": "1.9.0"
+    "@opentelemetry/api": "1.9.0",
+    "dotenv": "16.4.5"
   },
   "devDependencies": {
-    "@types/json-schema": "7.0.15",
     "@edge-runtime/vm": "^5.0.0",
+    "@types/json-schema": "7.0.15",
```

Comment:
pls revert these changes we cannot have dependencies from ai on those pkgs that is why we must use gateway